packages/-build-infra/.npmignore

```diff
@@ -0,0 +1,31 @@
+# compiled output
+/dist/
+/tmp/
+
+# dependencies
+/bower_components/
```

Comment:
Don't need to add this I think

Reply:
Also the other bower things

Reply:
opened a different PR to separate out the blueprint changes, also dropped these for you :)